docs/release-notes.md

```diff
@@ -8,7 +8,42 @@ toc_max_heading_level: 2
 
 # Release notes
 
-Welcome to Semgrep release notes. This document provides an overview of the changes, additions, and selected important fixes. Release notes published since April 2022 include Semgrep CLI, CI, and Semgrep App updates. Release notes published since April 2022 also include updates from more versions collected together.
+Welcome to Semgrep release notes. This document provides an overview of the changes, additions, and selected important fixes. Release notes published since April 2022 include updates for Semgrep command-line interface (CLI), Semgrep in continuous integration (CI), and Semgrep App updates. Release notes published since April 2022 also include updates from more versions collected together.
+
+## August 2022
+
+### Semgrep App
+
+#### Additions
+
+- Azure Pipelines CI configuration is now available when adding a new repository to Semgrep App for scanning. Users can select **Azure Pipelines** from within the App, and Semgrep generates a code snippet that users can copy and commit to their configuration file to set up their CI job.
+- Users can now delete projects in bulk (also known as batch delete) from Semgrep App's interface. To do this, sign into **Semgrep App** > **Projects**, and click on **Edit Projects**.
+- Users can now see usage limits in **Semgrep App** > **Settings**.
+
+### Semgrep CLI
+
+These release notes include upgrades for versions ranging between 0.108.0 and 0.111.0.
+
+#### Additions
+
+- Semgrep now provides experimental support for the **Swift** language. See all languages that Semgrep supports in [Supported languages](https://semgrep.dev/docs/supported-languages/).
+- Add configuration options for using the tree-sitter library installed anywhere on the system.
+- Metrics now include language-aggregated parse rates (files, bytes). The purpose of this is to continue with parsing improvements. See [Semgrep privacy policy](/metrics.md) for more details.
+- Semgrep CI now accepts more formats of Git URLs for metadata that are sent to semgrep.dev.
+
+#### Changes
+
+- Previously, the following error message appeared when metrics have not been uploaded within the set timeout timeframe:
+  ```
+  Error in send: HTTPSConnectionPool(host='metrics.semgrep.dev', port=443): Read timed out. (read timeout=3)
+  ```
+  As this caused confusion when running the CLI, this message is now displayed for development and debugging purposes only. Note that metrics are still successfully uploaded, but the success status is not sent in time for the current timeout set.
+
+- `semgrep ci` now defaults to fail open and always exits with exit code 0, which is equivalent to passing `--suppress-errors`. To disable this behavior, you can pass `--no-suppress-errors`, surfacing all exit codes to the CI provider. See [Configuring blocking findings and errors](/semgrep-ci/configuration-reference.md/#configuring-blocking-findings-and-errors) for more information.
```

Comment:
```suggestion
- `semgrep ci` now defaults to fail open on internal errors and always exits with exit code 0, which is equivalent to passing `--suppress-errors`. To disable this behavior, you can pass `--no-suppress-errors`, surfacing all exit codes to the CI provider. See [Configuring blocking findings and errors](/semgrep-ci/configuration-reference.md/#configuring-blocking-findings-and-errors) for more information.
```

Reply:
There are two possible fail opens in Semgrep, (fail open on findings, fail open on internal errors) so I just specified which one.

Reply:
Yeah, I think the term fail open is confusing by itself.